README.MD

```diff
@@ -106,6 +114,18 @@ You can also create reusable templates
     echo -n '{"text":"my phone number is 057-555-2323 and my credit card is 4961-2765-5327-5913", "AnalyzeTemplateId":"<my-analyze-template-name>", "AnonymizeTemplateId":"<my-anonymize-template-name>"  }' | http <api-service-address>/api/v1/projects/<my-project>/anonymize
     ```
 
+***Sample 5 (Image anonymization)***
+
+1. Create an anonymizer image template (This template redact values with black color)
+    ```sh
+    echo -n '{"fieldTypeGraphics":[{"graphic":{"fillColorValue":{"blue":0,"red":0,"green":0}}}]}' | http <api-service-address>/api/v1/templates/<my-project>/anonymize-image/<my-anonymize-image-template-name>
```

Comment:
In the code of the anonymizer if FillColorValue is empty you're setting the default to black.. 
consider adding an example here with empty FillColorValue

Reply:
maybe a different color.  I don't think putting empty values is a good practice